banco.sql

```diff
@@ -92,11 +92,12 @@ CREATE TABLE IF NOT EXISTS `lancamentos` (
   `categorias_id` INT NULL,
   `contas_id` INT NULL,
   `vendas_id` INT NULL,
-  `usuarios_id` INT NOT NULL,
+  `usuarios_id` INT NULL,
```

Comment:
Aqui pode ser um campo null? Acredito que o correto é not null mesmo, até porque pela a alteração o lançamento precisa de saber quem foi que alterou, num é isso?

Reply:
Verdade wil, sempre vai ter um usuário, eu testei rodando a db no workbanck, no phppmyadmin deu erro vou olhar aqui , se tiver alguma sugestão também